src/components/select/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import styles from './styles.module.scss'
+import ReactSelect from 'react-select'
+
+const style = {
+  option: (provided, state) => ({
+    ...provided,
+    color: state.isSelected ? 'var(--background-color)' : 'var(--text-color)',
+    backgroundColor: state.isSelected
+      ? 'var(--text-color)'
+      : 'var(--background-color)',
+  }),
+  valueContainer: (provided) => ({
+    ...provided,
+    padding: '0px',
+  }),
+  control: (provided, state) => ({
+    ...provided,
+    color: state.isSelected ? 'red' : 'var(--text-color)',
+    backgroundColor: 'var(--background-color)',
+    border: 'none',
+    // none of react-select's styles are passed to <Control />
+    // width: 200,
+  }),
+  menu: (provided, state) => ({
+    ...provided,
+    backgroundColor: 'var(--background-color)',
+  }),
+  clearIndicator: (provided, state) => ({
```

Comment:
can probably be removed, because no styles get changed

Reply:
Yes that's part of the leftovers I planned to remove, I'll do a pass with your edits and these later today 🚀